shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -785,6 +791,12 @@ object GlutenConfig {
       .intConf
       .createWithDefault(4096)
 
+  val SHUFFLE_WRITER_BUFFER_SIZE =
+    buildConf(GLUTEN_SHUFFLE_WRITER_BUFFER_SIZE)
+      .internal()
+      .intConf
+      .createWithDefault(4096)
+
```

Comment:
Can we by default use `spark.gluten.sql.columnar.maxBatchSize`? For backward compatibility.